modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -466,6 +475,9 @@ ComputeMultipleInelasticStress::computeAdmissibleState(unsigned model_number,
                                                        RankTwoTensor & inelastic_strain_increment,
                                                        RankFourTensor & consistent_tangent_operator)
 {
+  // Properly update material properties (necessary if substepping is employed).
```

Comment:
"properly update" is not very descriptive. Is it maybe "reset deformation state to the beginning of the increment" or something along those lines?

Reply:
Yeah, it's resetting material properties.